src/coreclr/pal/inc/pal.h

```diff
@@ -1531,6 +1531,14 @@ typedef struct _XMM_SAVE_AREA32 {
 
 typedef struct DECLSPEC_ALIGN(16) _CONTEXT {
 
+    _CONTEXT() = default;
```

Comment:
This change goes against the spirit of the Win32 PAL. The Win32 PAL is meant to mirror the Windows SDK definitions. The Windows SDK definitions of this structure is POD struct without any C++. This makes it C++ struct and behave differently from the Windows SDK definitions that can lead to all sort of surprises.

Reply:
@jkotas the proper fix in the spirit of the Win32 PAL would be to implement `CopyContext` (and maybe `InitializeContext` too) and modify the whole codebase to use that instead of pure assignments of the `CONTEXT` structure. That sounded like too extensive change given the number of places we just assign `CONTEXT` structs by value in the runtime and the time I had left before the RC1 snapshot.

I'd be ok reverting this PR though and going for the cleaner solution if you prefer.

Reply:
If you are ok with this, I am ok with it too. I wanted to point this difference out since it is likely going to take somebody by surprise at some point.

If you would like to clean it up, it can wait for .NET 9.

Reply:
Great, I'll leave this checked in and clean it up in 9 then.